ipfinder/metadata/ip_finder_metadata.go

```diff
@@ -0,0 +1,51 @@
+package metadata
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancher/go-rancher-metadata/metadata"
+	"time"
+)
+
+const (
+	metadataURL = "http://rancher-metadata/latest"
+	empty       = ""
```

Comment:
we should rename this param so it's use is obvious. It is a bit an overkill though to create a constant for an empty string :)